### PR TITLE
parallelize file uploads

### PR DESCRIPTION
**Pull request by ammardodin**

### CODE REVIEW

File: src/commands/upload.js [hunk 1]
```diff
@@ -114,9 +114,7 @@ export const handler = async (args) => {
 
   console.info(`Found ${fileList.length} file${fileList.length === 1 ? '' : 's'} ...`);
 
-  for (const fileInfo of fileList) {
-    await uploadFile(fileInfo);
-  }
+  await Promise.all(fileList.map(uploadFile));
```
Comment (pkriete) [inline src/commands/upload.js]: Can we chunk this? I don't think we want to do 100s of files in parallel in the worst case.

Comment (pkriete) [reply src/commands/upload.js]: Also, how does this affect error reporting? Are we going to spam the same error message if something is down?

Comment (ammardodin) [reply src/commands/upload.js]: How many do we want to do at once ? `10` ?

Comment (pkriete) [reply src/commands/upload.js]: Sounds reasonable to me.

Comment (ammardodin) [reply src/commands/upload.js]: I think each call will `console.error` the name of the file if it fails to upload.

**Pull request merged**